pouta_blueprints/static/partials/welcome.html

```diff
@@ -1,10 +1,43 @@
-<div class="jumbotron">
-    <h1><configurable-value key="INSTALLATION_NAME"></configurable-value></h1>
-    <p configurable-show key="ENABLE_SHIBBOLETH_LOGIN">
-        <b>You can also login with</b>
-        <a href="/login"><img src="/img/haka.gif" alt="Haka login" /></a>
-    </p>
-    <p>If you don't have access to your password, you
-    can <a href="#/reset_password">reset it here</a>.
-    </p>
+<div class="jumbotron" ng-controller="NavbarController">
+        <div class="col-md-6 col-xs-6">
```

Comment:
without `<div class="row">` just before line 2 , the view looks a bit odd, the haka login and normal login are out of place.